src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,31 @@
 import './ChatEntry.css';
+import TimeStamp from './TimeStamp';
+import propTypes from 'prop-types';
 
-const ChatEntry = () => {
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onToggleLike }) => {
+
+  const localOrRemote = sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
+    
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={localOrRemote}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">
+          <TimeStamp time={timeStamp} />
```

Comment:
Great job on interpolating your JS (props) into your JSX object return body and leveraging your `TimeStamp` component!